cf/api/resources/routes.go

```diff
@@ -15,8 +15,14 @@ type RouteEntity struct {
 	Space           SpaceResource           `json:"space"`
 	Apps            []ApplicationResource   `json:"apps"`
 	ServiceInstance ServiceInstanceResource `json:"service_instance"`
+	Options         map[string]string       `json:"options"`
 }
 
+/*type RouteOptions struct {
+	LoadBalancing string `json:"load_balancing"`
+	SessionCookie string `json:"session_cookie"`
+}*/
+
```

Comment:
This is left on purpose to trigger a discussion about the type formalization. 
Should we select a struct instead of a map? The map is currently selected, since it is easier to set a value of the map and we could add additional options easier. 
Option validation would be easier when using a struct.

Reply:
I'd stick to a map. We initially thought to add validation in the client to improve the UX but this will then require users to upgrade the CLI to make use of the new fields which seems unnecessary.

Reply:
Thanks, I would also prefer that at this moment.